### PR TITLE
Feature: Increase max task per node

### PR DESCRIPTION
**Pull request by timotheeguerin**
Hard coded to 4 for now because we need to know the number of cores per vm size which would require getting the vm size specs

#527

### CODE REVIEW

File: aztk/client.py [hunk 1]
```diff
@@ -108,6 +108,9 @@ def __create_pool_and_job(self, cluster_conf: models.ClusterConfiguration, softw
             start_task=start_task,
             enable_inter_node_communication=True if not cluster_conf.subnet_id else False,
             max_tasks_per_node=4,
+            task_scheduling_policy=batch_models.TaskSchedulingPolicy(
+                node_fill_type=batch_models.ComputeNodeFillType.pack,
```
Comment (paselem) [inline aztk/client.py]: Should we postpone until we know the service can handle this correctly?

Comment (jafreck) [reply aztk/client.py]: This is not a fix for anything, but it is better than what we have today. In the simple case, the underlying issue will not be present. Currently the second submitted application is not correctly handled. This delays the issue until the 5th concurrent application at least.

**Comment by darylmsft** [comment]
Is “4” always going to work?

d

**Comment by jafreck** [comment]
@darylmsft 4 will always work since the minimum number of cores is 1 and batch allows 4 tasks per core per vm.

**Pull request merged**